foursight_core/react/api/react_api.py

```diff
@@ -184,6 +184,14 @@ def _get_user_statuses(self, env: str) -> list:
             return []
         return [{"id": status, "name": status, "title": status.title()} for status in user_schema_status_enum]
 
+    @classmethod
+    def _is_test_name(cls, name):
+        return isinstance(name, str) and name.startswith("test_x")
+
+    @classmethod
+    def _is_test_name_item(cls, item, property_name = "name"):
```

Comment:
I'd probably have called it `_is_test_item`, but that's a small detail. This is fine.